internal/repository/meal_repository.go

```
@@ -11,172 +13,128 @@ import (
-    defer tx.Rollback()
-    
-		stmt, err := tx.Prepare(`
+	if len(menuItems) == 0 {
+		return nil
+	}
+
+	tx, err := r.db.Begin()
+	if err != nil {
+		return fmt.Errorf("failed to begin transaction: %w", err)
+	}
+	defer tx.Rollback()
```

golangci-lint (1.64.8)

Error return value of `tx.Rollback` is not checked

(errcheck)